cumulusci/tasks/bulkdata/load.py

```diff
@@ -83,6 +87,13 @@ def _init_options(self, kwargs):
         if self.bulk_mode and self.bulk_mode not in ["Serial", "Parallel"]:
             raise TaskOptionsError("bulk_mode must be either Serial or Parallel")
 
+        self.options["inject_namespaces"] = process_bool_arg(
+            self.options.get("inject_namespaces", False)
```

Comment:
What are the risks of injecting by default?

Reply:
That's a really good question. To be honest, I'm leery of it primarily because the implications are so hard to reason out, across complex scenarios like multi-package assemblages with unmanaged metadata in namespaced orgs.

It is probably safe. There's an additional check I want to add so that we don't attempt to namespace-inject a field if its namespace-injected version is also present in the file, to support the corner case where a field is present both namespaced and non-namespaced. That would make it even safer.

Reply:
In the current version, I have adopted on-by-default injection.